File: 3st/Jaeyoung/조건에 맞게 수열 변환하기 1.cpp
#include <string>
#include <iostream>
#include <vector>

using namespace std;

vector<int> solution(vector<int> arr) {
    vector<int> answer;
    for( auto num : arr ) {
        if( num >= 50 && num%2 == 0 ) {
            answer.push_back(num/2);
        } // if
        else if( num < 50 && num%2 == 1 ) {
            answer.push_back(num*2);
        } // else if
        else {
            answer.push_back(num);
        } // else
    } // for
    return answer;
} // solution

File: 3st/Hyeonseo/programmer/조건에 맞게 수열 변환하기 1.py
def solution(arr):
    answer = []
    for num in arr:
        if num >= 50 and num % 2 == 0:   # 50 이상 짝수 → 2로 나누기
            answer.append(num // 2)
        elif num < 50 and num % 2 == 1:  # 50 미만 홀수 → 2 곱하기
            answer.append(num * 2)
        else:                            # 그 외는 그대로
            answer.append(num)
    return answer
